src/Entries/EntryQueryBuilder.php

```diff
@@ -33,17 +33,20 @@ protected function column($column)
             return $column;
         }
 
+        $table = Str::contains($column, '.') ? Str::before($column, '.') : '';
+        $column = Str::after($column, '.');
+
         if ($column == 'origin') {
             $column = 'origin_id';
         }
 
         if (! in_array($column, self::COLUMNS)) {
-            if (! Str::startsWith($column, 'data->')) {
+            if (! Str::contains($column, 'data->')) {
```

Comment:
The test still passes if I revert this back to `startsWith`. Is it necessary to change it?

Reply:
Yeah should be starts with - it’s a better approach

Reply:
I agree. It seems like maybe the `contains` check just stuck around accidentally while you figured out the solution to the issue?

Reply:
Yeah - contains could potentially return false results too. Please change it :)